holidays/groups/christian.py

```diff
@@ -299,10 +315,34 @@ def _add_epiphany_day(self, name, calendar=None) -> date:
         calendar = calendar or self.__calendar
         self.__verify_calendar(calendar)
 
-        return (
-            self._add_holiday_jan_19(name)
-            if self.__is_julian_calendar(calendar)
-            else self._add_holiday_jan_6(name)
+        if self.__is_julian_calendar(calendar):
+            return self._add_holiday(
+                name, _timedelta(date(self._year, JAN, 19), julian_calendar_drift(self._year - 1))
+            )
+        elif self.__is_ethiopian_calendar(calendar):
+            return self._add_holiday(
+                name,
+                _timedelta(date(self._year, JAN, 20), julian_calendar_drift(self._year - 1))
+                if is_ethiopian_leap_year(self._year, is_previous=True)
```

Comment:
If we use the expression `self._year - 1` for `julian_calendar_drift`, can we do the same for `is_ethiopian_leap_year`, and it will be simpler than a parameter?